packages/swingset-liveslots/tools/fakeVirtualSupport.js

```diff
@@ -348,6 +356,7 @@ export function makeFakeVirtualStuff(options = {}) {
   vom.initializeKindHandleKind();
   const cm = makeFakeCollectionManager(vrm, fakeStuff, actualOptions);
   const wpm = makeFakeWatchedPromiseManager(vrm, vom, cm, fakeStuff);
+  wpm.preparePromiseWatcherTables();
```

Comment:
I remain suspicious that this call should not be here.  However, it is normally done from `startVat`, which I suppose is always going to be absent from tests using the fake virtual stuff, so providing the call here is certainly a convenience.  Just beware that this might come back later to bite us.